modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json

```diff
@@ -836,6 +836,17 @@
           "23", "24", "25", "26", "27", "28", "Z", "LG34", "LG35", "LGE22"
       ]
     },
+    {
+      "id": "sander_lucioperca_EVA_PRJEB42376",
+      "source_name": "PRJEB42376",
```

Comment:
On the EVA page the study PRJEB42376 is for the assembly GCA_008315115.2
https://www.ebi.ac.uk/eva/?eva-study=PRJEB42376